src/components/organisms/TodoListComponent.tsx

```diff
@@ -50,6 +51,7 @@ export const TodoListComponent = () => {
             .filter((id) => id !== null)
         : todoOrder,
     );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [scheduleReadResult]);
```

Comment:
```suggestion
            .filter((id) => id !== null)
        : [],
    );
  }, [scheduleReadResult]);
```

↑これでも良いですか？
useEffect内でtodoOrderを使っている意味はあまりなさそうなので！
お手数ですが、お願いします :bow: